arm/Microsoft.ServiceFabric/clusters/.bicep/nested_applications.bicep

```diff
@@ -0,0 +1,24 @@
+param applicationObj object
+param clusterName string
+param location string
+param tags object
+param identity object
+param properties object
+
+resource application 'Microsoft.ServiceFabric/clusters/applications@2021-06-01' = {
+  name: '${clusterName}/application-${applicationObj.name}'
+  location: location
+  tags: tags
+  identity: identity
+  properties: properties
+
+  resource services 'services@2021-06-01' = [for service in applicationObj.applicationsServices: {
```

Comment:
Must be refactored to a child resource folder & template. See for example storage account from upstream main for reference

Reply:
Refactored.